rest-service/manager_rest/resource_manager.py

```diff
@@ -992,7 +992,6 @@ def execute_workflow(self, execution, *, force=False, queue=False,
                         DeploymentState.INACTIVE
                 execution.deployment.deployment_status =\
                     DeploymentState.IN_PROGRESS
```

Comment:
wondering if those lines should go as well

Reply:
Where are they currently handled other than here?

Reply:
resource_manager's `update_deployment_statuses` called from `update_execution_status`
ie. when the execution is set to started